Makefile

```diff
@@ -181,7 +181,7 @@ start-e2e: cli
 	# check we can connect to Docker to start Redis
 	docker version
 	kubectl create ns argocd-e2e || true
-	kubectl config set-context --current --namespace=argocd-e2e
+	kubens argocd-e2e
```

Comment:
I think there was a reason for this.

Reply:
@alexmt?

Reply:
`kubens [NS]` seems to be the same as `kubectl config set-context --current --namespace=[NS]`:

https://github.com/ahmetb/kubectx/blob/543e03509019dae0edacf559ee6b06fb904a046a/kubens#L93

Reply:
I'm going to suggest you leave it as is then.

Reply:
Sounds good to me, I'll take this change out